test/hack/resource/go.mod

```diff
@@ -1,48 +1,50 @@
 module github.com/aws/karpenter-provider-aws/test/hack/resource
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.2
```

Comment:
Do we need this toolchain command or can we directly pin to the patch version in the `go` block

Reply:
Removed.